api4/emoji.go

```diff
@@ -22,6 +22,7 @@ func InitEmoji() {
 	BaseRoutes.Emojis.Handle("", ApiSessionRequired(getEmojiList)).Methods("GET")
 	BaseRoutes.Emoji.Handle("", ApiSessionRequired(deleteEmoji)).Methods("DELETE")
 	BaseRoutes.Emoji.Handle("", ApiSessionRequired(getEmoji)).Methods("GET")
+	BaseRoutes.Emoji.Handle("/image", ApiSessionRequired(getEmojiImage)).Methods("GET")
```

Comment:
Can we make this `ApiSessionRequiredTrustRequester`? That way the browser can request it